builtin/mainmenu/tab_settings.lua

```diff
@@ -59,6 +59,30 @@ local function getMipmapSettingIndex()
 	return 1
 end
 
+local display_resolution = {
+	{"800 x 600 (4:3),960 x 540 (16:9),1024 x 768 (4:3),1152 x 864 (4:3),1280 x 720 (16:9)"},
```

Comment:
*1152x864* isn't a common resolutions.
Other resolutions like *1920x1080* or (maybe) *2560x1440* are missing.
More generally, hardcoding resolutions is bad and the main menu should preset some options based on the videomodes Irrlicht supports.